styles/style_guide/HeadingPunctuation.txt

```diff
@@ -0,0 +1,50 @@
+# Need to update this for "1. Click this" "1.1. Click that"
```

Comment:
I feel similarly about this rule and the oxford comma rule. Both create false positives. Rather than fiddle with the regex, I would rather find a way to keep these rules so we can run them manually from time to time to check if there are any more errors popping up, but without including them in whatever CI pipeline we eventually build 👌🏻 

As it is now the file is a bit confusing with two separate rules next to one another. Could you clean it up a bit? I also think that `level` should be `suggestion` for any rule that leads to false positives.

I imagine that there is a better way to tell Vale to ignore this file besides commenting out every line 🕵🏻